.env.example

```diff
@@ -1,2 +1,3 @@
 MEILISEARCH_HOST=http://0.0.0.0:7700
 MEILISEARCH_API_KEY=searchKey
+NEXT_PUBLIC_FATHOM_TRACKING_CODE=
```

Comment:
I think there's no need to add it here since the `.env.example` contains the minimum recommended for the application to work :) Just ensure that the app works fine without it, and that should be fine!